### ISSUE TITLE
The keyboard requires an additional stroke to move to the next control from the Graph: A11y_.NET 9 template_.NET 9 template_Project_FocusOrder.

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/maui#26127]
### GitHub Tags
#A11yTCS; #A11ySev3; #Win32; #WCAG2.4.3; #GH_.NET9template_Win32_Apr2024; #DesktopApp; #.NET Core; #Keyboard; #A11yMAS; #FocusOrder;

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
OS: Windows 11 Enterprise 24H2 (OS build 26100.2314)

### Repro Steps:

1. Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Tab till projects section.
4. Observe that the keyboard requires an additional stroke to move to the next control from the Graph.

### Actual Result:
The keyboard requires an additional stroke to move to the next control from the Graph.

### Expected Result:
After Graph by pressing tab, the focus should be going to the Projects option link work.

### User Impact:
keyboard users will not be able to access the controls.

### Attachment:
https://github.com/user-attachments/assets/ffecf111-f23e-45ea-8b2e-c740524e8224

**Comment by AnKushSingh05** [comment]
Issue is not fixed. Please find attachment for reference

https://github.com/user-attachments/assets/cf14ac03-1291-49a8-89d8-84d2f70c5ab7

**Comment by AnKushSingh05** [comment]
#Closed;#A11yFixed;
Issue is fixed

https://github.com/user-attachments/assets/0924d715-b79a-4716-b884-eb2e739b3f86
